Datacollection: Fix file (inc. media) record upload changes and deletions

iszmais opened the pull request:
Fixes general Issues within the saving, editing and deleting of data fields (File and Media).

https://mantis.ilias.de/view.php?id=40975
https://mantis.ilias.de/view.php?id=35639

iszmais committed:
Fix file (inc. media) record upload changes and deletions

oliversamoila commented:
Hello Ingmar. 
please replace the Search glyph in https://github.com/ILIAS-eLearning/ILIAS/pull/8809/files#diff-122aed9d180c14db6823779f7057ce6521776c048f172bc432ceeeaf279ea335R266 with the Enlarge glyph. 
Hopefully we'll keep it for the longer term. 😉

Search » https://test10.ilias.de/go/stys/21/SymbolGlyphGlyphSearch/default/delos
Enlarge » https://test10.ilias.de/go/stys/21/SymbolGlyphGlyphEnlarge/default/delos

Thank you @yvseiler. 
Thank you @iszmais.

Best regards
Oliver

iszmais commented:
@oliversamoila thanks for the observation!

I fixed this partially: https://github.com/ILIAS-eLearning/ILIAS/pull/8838
During the change I discovered that there is no `enlarge` glyph for ILIAS 9 (which might have caused my initial confusion).
Therefore, I implemented the requested change only for ILIAS 10+. Therefore, we currently have three states:
- ILIAS 10+: Enlarge Glyph
- ILIAS 9: Search Glyph
- ILIAS 8 (deprecated): Legacy Search Icon

I'm not sure if this is intentional an ILIAS 10 glyph or just forgotten. If it's downgraded to ILIAS 9 I can adapt it. However, I'm fine with both solutions.

Greetings,
@iszmais

oliversamoila commented:
Thank you @iszmais for editing the carousel of intended symbols. 
Sorry, I was not able to get this in time during the various phases of other changes.

It remains in the way you just described. This is also an acceptable transition for users for a uniform visual language. 
Backporting the glyph from 10 is not intended after consultation.

Thank you very much.
Best regards
@oliversamoila
